client/src/components/Account.jsx

```diff
@@ -50,6 +50,7 @@ export default class Account extends React.PureComponent {
         profileTab: 'active',
         passwordTab: '',
         postsTab: '',
+        followTab: '',
     }
 
     state = {... this.initialState}
```

Comment:
i think you need to remove the space after the spread operator
...this.initialState